src/engine/instrument.h

```diff
@@ -457,10 +474,8 @@ struct DivInstrumentWaveSynth {
 };
 
 struct DivInstrumentSoundUnit {
-  bool useSample;
```

Comment:
Why?

Reply:
it's for make useSample flag "global" for every chip w/ sample support